Makefile

```diff
@@ -89,3 +89,6 @@ uploadmessages:
 
 downloadmessages:
 	java -jar crowdin-cli.jar download -b `git symbolic-ref HEAD | xargs basename`
+
+distributefrontendmessages:
+	python ./kolibri/utils/distribute_frontend_messages.py
```

Comment:
No, I don't. Happy to move it elsewhere, just not sure where 'build' utils live :)